modules/migration-known-issues.adoc

```diff
@@ -31,3 +31,7 @@ ifdef::migrating-3-4[]
 
 * If you cannot install CAM 1.2 on an {product-title} 3 cluster, download the current `operator.yml` file, which fixes this problem. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1843059[*BZ#1843059*])
 endif::[]
+
+ifdef::migrating-4-2-4[]
+* In the current release (CAM 1.2), you cannot migrate a namespace that contains imagestreams from OpenShift Container Platform 4.4 or later. An error message similar to the following appears in the Velero log: `Error restoring nametags`. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1848561[*BZ#1848561*])
```

Comment:
Should also update the lead-in sentence. You can change to "The following error message is displayed [we try to avoid 'appears'] in the Velero log:"